base/multidimensional.jl

```diff
@@ -51,10 +51,14 @@ module IteratorsMD
 
     # arithmetic, min/max
     (-){N}(index::CartesianIndex{N}) = CartesianIndex{N}(map(-, index.I))
-    (+){N}(index1::CartesianIndex{N}, index2::CartesianIndex{N}) = CartesianIndex{N}(map(+, index1.I, index2.I))
```

Comment:
Do these definitions substantially exceed the line length recommendation? These eyes find the existing (aligned) version easier to parse as a whole than the line-split version. Best!